### ISSUE TITLE
[Feature] Expose all aria attributes

### ISSUE THREAD

**Issue by lemnis** [microsoft/playwright#14332]
Currently, `accessibilty.snapshot` exposes a subset of aria attributes while each dev tool does expose it. E.g. `aria-current` (see screenshots). Exposes them would help in improving a11y testing.
![Screenshot from 2022-05-22 09-47-31](https://user-images.githubusercontent.com/3999815/169685013-07176107-bf3c-46f3-8acc-946fd6fb5cb6.png)
![Screenshot from 2022-05-22 09-43-48](https://user-images.githubusercontent.com/3999815/169685014-be706811-a3de-4f30-b494-47ae45b9f0fb.png)

![Screenshot from 2022-05-22 09-49-03](https://user-images.githubusercontent.com/3999815/169685011-81903b5e-1d51-4bef-838c-470c8c9b0f4b.png)

Currently missing:

-   atomic
-   busy
-   colcount
-   colindex
-   colspan
-   current
-   dropeffect
-   grabbed
-   live
-   placeholder
-   posinset
-   relevant
-   rowcount
-   rowindex
-   rowspan
-   setsize
-   sort
- controls
- details
- errormessage
- flowto

(I didn't include a couple of attributes as they already exposed in some other format, e.g. `hidden` will correspond with `ignored`)

**Comment by tylerthehaas** [comment]
Has there been any further thought on this issue? We are currently in need of support for this feature in our testing sweet.

**Comment by weikinhuang** [comment]
I'm mostly interested in `busy` being implemented (https://testing-library.com/docs/queries/byrole#busy) as this is needed for lots of parts of a dynamic website that have mounting after SSR returns the un-hydrated html.

A different suggestion might be to add a `is` filter on the locators ie. `getByRole('navigation', { is: '[aria-busy="false"]' })` or via `.filter()`

**Comment by pavelfeldman** [comment]
I don't think we are comfortable with bulk-implementing those, but are open to explicit requests. Please file an issue per aria property with the example and a use case. Adding each property will affect both role locators as well as aria snapshots, so we need to handle it carefully, bit by bit.
